src/components/GenreMenu.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function GenreMenu(props) {
+    return (
+        <menu className='genre-menu'>
+            {props.genres.map((g, i) => <li className={i === props.value ? 'genre genre--chosen' : 'genre'}>
```

Comment:
It's better to use meaningful names like

```suggestion
            {props.genres.map((genre, i) => <li className={i === props.value ? 'genre genre--chosen' : 'genre'}>
```